docs/rules/no-promise-reject.md

```diff
@@ -34,12 +36,6 @@ Examples of **correct** code for this rule:
 async function divide(x, y) {
   const [xv, yv] = await Promise.all([x, y]);
 
-  return yv === 0
-    ? new Error("Cannot divide by zero.")
-    : xv / yv;
+  return yv === 0 ? new Error("Cannot divide by zero.") : xv / yv;
 }
 ```
-
-## Options
```

Comment:
`Options` sections are removed when a rule does not have options. This is part of the [`--rule-doc-section-options`](https://github.com/bmish/eslint-doc-generator#configuration-options) option.